src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -2592,6 +2592,11 @@ def get_access_results_urls(
     """
     Endpoint for retrieving access results URLs for a privacy request.
     """
+    if not CONFIG.security.subject_request_download_ui_enabled:
+        raise HTTPException(
+            status_code=HTTP_404_NOT_FOUND,
```

Comment:
nit: I think a 403 might be more correct in this case because it indicates that the user doesn't have access to it
```suggestion
            status_code=HTTP_403_FORBIDDEN,
```